# Written and placed in public domain by Jeffrey Walton.
# This patch fixes some issues with cURL.
--- libcurl.pc.in
+++ libcurl.pc.in
@@ -25,6 +25,7 @@
 #
 prefix=@prefix@
 exec_prefix=@exec_prefix@
+bindir=@bindir@
 libdir=@libdir@
 includedir=@includedir@
 supported_protocols="@SUPPORT_PROTOCOLS@"
